x-pack/plugins/ml/server/lib/route_guard.ts

```diff
@@ -26,14 +26,14 @@ type MLRequestHandlerContext = RequestHandlerContext & {
   alerting?: AlertingApiRequestHandlerContext;
 };
 
-type Handler = (handlerParams: {
+type Handler<P = unknown, Q = unknown, B = unknown> = (handlerParams: {
   client: IScopedClusterClient;
-  request: KibanaRequest<any, any, any, any>;
+  request: KibanaRequest<P, Q, B, any>;
```

Comment:
the 4th generic in `KibanaRequest` is `Method extends RouteMethod = any` so we can omit it 
```suggestion
  request: KibanaRequest<P, Q, B>;
```